ledger/src/blockstore_db.rs

```diff
@@ -405,9 +502,13 @@ pub trait Column {
 
     fn key(index: Self::Index) -> Vec<u8>;
     fn index(key: &[u8]) -> Self::Index;
-    fn primary_index(index: Self::Index) -> Slot;
```

Comment:
So, `Slot` was misnomer in this instance. I tricked a bit about this. And I need to read the code around TransactionStatusIndex...